hhReweight/MakeMap.cpp

```diff
@@ -103,21 +103,22 @@ int main ()
     // TH2F* hMapFolded = new TH2F ("allHHNodeMapFolded", "allHHNodeMapFolded", 90, 0, 1800, 5, 0, 1); // won't be used for reweight
     // TH1F* hMap1D = new TH1F ("allHHNodeMap1D", "allHHNodeMap1D", nbins_mHH, binning_mHH);
 
-    // new binning for dynamic reweight -- again these weird Xanda binnings
-    TFile* fOut = new TFile ("outMap_5Dbinning.root", "recreate");
-    double binning_mHH [56] = { 250,260,270,280,290,300,310,320,330,340,
-                                350,360,370,380,390,400,410,420,430,440, 
-                                450,460,470,480,490,
-                                500,510,520,530,540,550,600,610,620,630,
-                                640,650,660,670,680,690,700,750,800,850,
-                                900,950,1000,1100,1200,1300,1400,1500.,1750,2000,50000};
+    // new binning for differential reweight -- again these weird Xanda binnings
```

Comment:
since the binning changed please remove the comment `-- again these weird Xanda binnings` :)

Reply:
It was not my comment but I liked it and kept it. I will be sad in not having it anymore haha